### ISSUE TITLE
Metrics graphs not drawing

### ISSUE THREAD

**Issue by sdesse** [ElasticHQ/elasticsearch-HQ#506]
### General information

- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Debian 9
- **ElasticHQ Version**: 3.5.0
- **Elasticsearch Version**: 6.8.4
- **Python version (ignore is using docker image)**: Python 3.4.2
- **Browser Vendor and Version (if applicable)**: Any Chrome, FF, Edge

### Issue Description

We run elasticsearchHD behing a reverse proxy.

The app is running on a monitoring host http://monitoringhost:4000on the internal network and it's published by a reverse proxy on Internet under the URL: https://proxy/no-access/elastichq/

The application is working fine but the the metrics graphs are not drawing.

It seems that when pooling the data for the graphs the path is not reflected in the calls:
Calls are: https://proxy/socket.io/?EIO=3&transport=polling&t=MxksoOM
Instead of https://proxy/no-access/elastichq/socket.io/?EIO=3&transport=polling&t=MxksoOM

If entering the correct path that can be retreinved.
I'm not a develloper and could not find where in the code to add the path to the URL?

### Source Code / Logs

See screenshots.

### Be Patient

Thanks for all your efforts.

**Comment by sdesse** [comment]
![elastichq](https://user-images.githubusercontent.com/44860623/70508414-5fbd9f00-1b69-11ea-9078-17074f7ddeef.JPG)

**Comment by markkutikkanen** [comment]
I have the same issue still on version 3.5.12

**Comment by sdesse** [comment]
Yes, I have upgraded to the latest hoping that it would be solved but not.

I'm ready to pay a little to get this fixed and few other cutomizations but not sure how to get this organised. If any suggestions please let me know.

**Comment by sdesse** [comment]
This is still no working (3.5.12).

I tried to see if I can fix this on my end with the reverse proxy but no luck. The setup is:

Web Client -> Internet -> mod_proxy -> ElasticHQ -> elasticcsearch cluster 

In the mod_proxy conf:
ProxyPass "/elastichq/" "http://127.0.0.1:5000/"
ProxyPassReverse "/elastichq/" "http://127.0.0.1:5000/"

All works well except graphs.

I  wish we could get this fixed and I am available for a debug session if it can help.
I even offered to pay for this fix and few other improvements as I don't code myself.

Hope this gets addressed in 2021, thanks!

**Comment by autokilla47** [comment]
April 2023
still...
